src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -118,21 +119,30 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     String conversationTitle = request.getParameter("conversationTitle");
+    String conversationUser = request.getParameter("conversationUser");
+    User userAdded = userStore.getUser(conversationUser);
     if (!conversationTitle.matches("[\\w]*")) {
       request.setAttribute("error", "Please enter only letters and numbers.");
       request.getRequestDispatcher("/WEB-INF/view/conversations.jsp").forward(request, response);
       return;
     }
 
-    if (conversationStore.isTitleTaken(conversationTitle)) {
+    if (conversationStore.isTitleTaken(conversationTitle) && userAdded != null) {
       // conversation title is already taken, just go into that conversation instead of creating a
-      // new one
+      // new one unless the entered user does not exist
       response.sendRedirect("/chat/" + conversationTitle);
       return;
     }
 
+    //Checks whether or not the user being added to the conversation exists
+    if(userAdded == null){
```

Comment:
We can move this check before the previous one so we don't have to check whether the user is null twice.

Reply:
Nice catch! Done :)